### PR TITLE
Bug fixing

### PR DESCRIPTION
**Pull request by edsonarios**

### CODE REVIEW

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 1]
```diff
@@ -15,11 +15,9 @@
 import org.fundacion.jala.converter.view.controllers.ClientRequest;
 import org.fundacion.jala.converter.view.Models.AudioRequestForm;
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
-import javax.swing.JPanel;
-import javax.swing.JButton;
-import javax.swing.BoxLayout;
-import javax.swing.JOptionPane;
-import java.awt.Font;
+
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: This space is unnecesary

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 2]
```diff
@@ -138,4 +149,19 @@ private void callRequest() throws IOException {
...
+            new Thread(this).start();
+        }
+        public void run() {
+
+            try {
+                callRequest();
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: This spaces are unnecesary

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 3]
```diff
@@ -11,9 +11,8 @@
 package org.fundacion.jala.converter.view.converter;
 
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
-import javax.swing.JComboBox;
-import javax.swing.JPanel;
-import javax.swing.JCheckBox;
+
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java]: Shouldnt be spaces betwen imports

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 4]
```diff
@@ -15,10 +15,8 @@
 import org.fundacion.jala.converter.view.Models.VideoRequestForm;
 import org.fundacion.jala.converter.view.controllers.ClientRequest;
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
-import javax.swing.JPanel;
-import javax.swing.JButton;
-import javax.swing.BoxLayout;
-import javax.swing.JOptionPane;
+
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: Space

File: src/main/java/org/fundacion/jala/converter/core/facade/ZipFileFacade.java [hunk 5]
```diff
@@ -10,6 +10,8 @@
  */
 package org.fundacion.jala.converter.core.facade;
 
+import org.fundacion.jala.converter.core.RunCommand;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/core/facade/ZipFileFacade.java]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 6]
```diff
@@ -138,4 +149,19 @@ private void callRequest() throws IOException {
         }
         LOGGER.info("Finish");
     }
+    public class _charge implements Runnable {
+        public void show() {
+            new Thread(this).start();
+        }
+        public void run() {
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 7]
```diff
@@ -48,6 +46,8 @@
     private final int fontSize = 12;
     private String token;
     private String checksumLocal;
+    private JLabel label;
+
```
Comment (cristianfchq) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java:55]: this line should be removed

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 8]
```diff
@@ -15,11 +15,9 @@
 import org.fundacion.jala.converter.view.controllers.ClientRequest;
 import org.fundacion.jala.converter.view.Models.AudioRequestForm;
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
-import javax.swing.JPanel;
-import javax.swing.JButton;
-import javax.swing.BoxLayout;
-import javax.swing.JOptionPane;
-import java.awt.Font;
+
+import javax.swing.*;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: You shouldn't import the entire library, just what you use

File: src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java [hunk 9]
```diff
@@ -15,11 +15,9 @@
...
+import java.awt.*;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/AudioConverterInterface.java]: You shouldn't import the entire library, just what you use

File: src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java [hunk 10]
```diff
@@ -11,9 +11,8 @@
...
+import javax.swing.*;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/OutputSettingsAudio.java]: You shouldn't import the entire library, just what you use.

File: src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java [hunk 11]
```diff
@@ -15,10 +15,8 @@
 import org.fundacion.jala.converter.view.Models.VideoRequestForm;
 import org.fundacion.jala.converter.view.controllers.ClientRequest;
 import org.fundacion.jala.converter.view.utilities.JLabelStyle;
-import javax.swing.JPanel;
-import javax.swing.JButton;
-import javax.swing.BoxLayout;
-import javax.swing.JOptionPane;
+
+import javax.swing.*;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/converter/VideoConverterInterface.java]: You shouldn't import the entire library, just what you use

File: src/main/java/org/fundacion/jala/converter/view/utilities/CheckFile.java [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+     */
+    public static boolean checkFileSelect(final String file) {
+        if (file != null) {
+            return true;
+        } else {
+            JOptionPane.showMessageDialog(null, "You need select a file", "Error Message", JOptionPane.ERROR_MESSAGE);
+            return false;
+        }
+    }
+}
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/utilities/CheckFile.java:34]: Add a space at the end of the class

File: src/main/java/org/fundacion/jala/converter/view/utilities/CheckFile.java [hunk 13]
```diff
@@ -0,0 +1,34 @@
...
+
+import javax.swing.JOptionPane;
+
+/**
+ * This class verify si is selected a file.
+ */
+public class CheckFile {
+
+    /**
+     * Verify that a file is selected
```
Comment (JorgeRCV) [inline src/main/java/org/fundacion/jala/converter/view/utilities/CheckFile.java]: javadoc has to end with a dot

**Review by JorgeRCV — APPROVED** [review]

File: src/main/java/org/fundacion/jala/converter/view/text_extractor/TextExtractorInterface.java [hunk 14]
```diff
@@ -26,6 +26,7 @@
 import java.io.IOException;
```
Comment (danielaSantaCruz) [inline src/main/java/org/fundacion/jala/converter/view/text_extractor/TextExtractorInterface.java]: This blank space should be deleted

**Review by danielaSantaCruz — APPROVED** [review]

**Review by cristianfchq — APPROVED** [review]

**Pull request merged**